<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyAVLTree</class>
 <widget class="QWidget" name="MyAVLTree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="deleteNode">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>295</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchNode">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>365</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索数据</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>15</y>
     <width>570</width>
     <height>160</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInit">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>25</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>915</x>
     <y>605</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="divTree">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>425</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>分裂树</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>185</y>
     <width>781</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addNode">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>235</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>插入数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mergerTree">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>485</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>合并树</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimeControl">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>600</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>速度调节 ：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
